pontoon/base/views.py

```diff
@@ -262,8 +262,25 @@ def _get_paginated_entities(locale, project, form, entities):
             if entity_pk in entities.values_list('pk', flat=True):
                 entities_to_map = list(entities_to_map) + list(entities.filter(pk=entity_pk))
 
+    entities = Entity.map_entities(locale, entities_to_map, [])
+    entities_updated = []
+
+    for entity in entities:
+        if entity['for_dummy']:
+            t = entity['translation']
+            print(t[0]['user'])
+            if t[0]['user'] not in [request.user.pk, -1]:
+                entity['translation'] = [{
+                    'fuzzy': False,
+                    'string': None,
+                    'approved': False,
+                    'pk': None,
+                    'user': -1,
+                }]
+        entities_updated.append(entity)
```

Comment:
I don't understand this. What is it supposed to do?

Reply:
Will add comments in next commit  Sorry for that. This part is supposed to remove translations of entities connected to the dummy project made by other users. If no translation is present for an entity still a translation goes in this format evident from this part https://github.com/VishalCR7/pontoon/blob/master/pontoon/base/models.py#L2109. So if another user's translation comes into query this part removes it and replaces it with a dummy one.